LondonBritishLibrary/orient/BLorient757.xml

```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">142</measure>
-                                    
-                                    
-                                    
-                                    
-                                    <dimensions type="outer" unit="mm">
-                                        <height>178</height>
-                                        <width>159</width>
+
+
+                                    <dimensions type="outer" unit="in">
+                                        <height>7</height>
+                                        <width>6.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="20 22" cert="high">
-                           <note>except <locus from="134" to="142"/> where the number is larger.</note>
+                                <layout columns="2" writtenLines="20 22">
+                           <note>On <locus from="134" to="142"/> the number of
+                           written lines is larger.</note>
                                 </layout>
                             </layoutDesc>
-                            
-                            
+
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is fairly written of the <date>18th century.</date>.
+                                <desc>The manuscript is fairly written.
```

Comment:
"fairly written" is a very old style expression. "Finely written"?

Reply:
I would not correct Wright's English unless truly necessary...